ContractExamples/scripts/setter-populate.sh

```diff
@@ -27,6 +27,8 @@ soroban contract build
 soroban contract deploy --wasm target/wasm32-unknown-unknown/release/setter.wasm \
       --source $ACCOUNT --network $NET | tee >.setter.id
 
+soroban contract invoke --id $(cat .setter.id) --source $ACCOUNT --network $NET -- init
```

Comment:
Why is this necessary?  All of the others overwrite these fields anyway.

Reply:
As I explained in the call yesterday, unassigned contract fields will cause `apalache-mc check` to fail, so you want to make sure that all fields exist from the beginning.